vcluster/deploy/topologies/air-gapped.mdx

```diff
@@ -0,0 +1,180 @@
+The `vcluster.yaml` file holds your vCluster configuration and allows overriding the vCluster Helm chart default values.
+
+The following are some default values that may need to be overridden for a vCluster Pro air-gapped install:
+
+### Set default image registry
+
+Use `defaultImageRegistry` to configure a private image registry.
+
+```yaml title="Default Image Registry configuration"
+controlPlane:
+  advanced:
+    defaultImageRegistry: my-private-registry:5000/vcluster/
+```
+
+vCluster prepends the image registry to all images used by vCluster, such as syncer and Kubernetes. For example, `registry.k8s.io/kube-apiserver:v1.30.2` becomes `my-private-registry:5000/vcluster/kube-apiserver:v1.30.2`.
+
+### Create image pull secret
+
+`imagePullSecrets` defines extra image pull secrets for the vCluster control plane `ServiceAccount`.
+
+```yaml title="Image Pull Secrets configuration"
+controlPlane:
+  advanced:
+    serviceAccount:
+      imagePullSecrets:
+        - name-of-your-image-pull-secret
+```
+
+### Configure API key
+
+The `apiKey` provides a way to deploy vCluster with Pro features without the platform agent installed on the host cluster.
+
+```yaml title="API Key configuration"
+external:
+  platform:
+    apiKey:
+      # Defaults to vcluster-platform-api-key if undefined
+      # The air-gapped license key provided by Loft has to be the data
+      # under the first key (name does not matter) of the secret.
+      secretName:
+      # Namespace to search for the secret. If undefined,
+      # it searches the namespace that the vCluster is deployed in.
+      # If different than the namespace of the vCluster deployment,
+      # then vCluster needs access to that namespace.
+      namespace:
+      # Default enabled to create the necessary RBAC roles
+      # and role bindings in order to find the secret
+      createRBAC: true
+```
+
+## Deploy vCluster Pro
+
+Create a Kubernetes `Secret` from the *vcluster-pro-air-gapped-license.txt* file provided by Loft in the `Namespace` where you are installing the air-gapped vCluster instance (see above if not creating `secret` in the vCluster install `namespace`):
+
+```bash title="Create Kubernetes Secret"
+kubectl create namespace vcluster-ns
+kubectl create secret generic vcluster-platform-api-key -n vcluster-ns \
+    --from-file=license-key=./v*cluster-pro-air-gapped-license.txt*
+```
+
+Create a `vcluster.yaml` file (Helm chart values file) based on the configuration examples from above:
+
+```yaml title="vcluster.yaml configuration"
+controlPlane:
+  advanced:
+    defaultImageRegistry: my-private-registry:5000/vcluster/
+    serviceAccount:
+      imagePullSecrets:
+        - name-of-your-image-pull-secret
+  backingStore:
+    etcd:
+      embedded:
+        enabled: true
+  coredns:
+    embedded: true
+
+external:
+  platform:
+    apiKey:
+      secretName: vcluster-platform-api-key
+```
+
+Use Helm to install the vCluster Pro instance into the `namespace` where you installed:
```

Comment:
Use Helm to install the vCluster Pro instance into the `namespace` where you installed the license `secret`: